jobs/client.go

```diff
@@ -0,0 +1,85 @@
+package jobs
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"log"
+	"net/http"
+	urlUtils "net/url"
+	"strconv"
+)
+
+const (
+	parserError        = "could not parse error"
+	unauthorizedAccess = "unauthorized access"
+	urlParseError      = "could not parse base url"
+)
```

Comment:
could you put the errors onto `errors.go` file

Reply:
Done